.github/workflows/preview.yml

```diff
@@ -0,0 +1,106 @@
+name: "Preview build"
+on:
+  pull_request:
+jobs:
+  build:
+    name: "Build preview"
+    runs-on: "ubuntu-latest"
+    steps:
+    - name: "Checkout repo"
+      uses: actions/checkout@v2
+      with:
+        submodules: recursive
+    - name: Comment on PR
+      run: |
+        set -euo pipefail
+        PR_ID=$(echo $GITHUB_REF | awk 'BEGIN { FS = "/" } ; { print $3 }')
+        echo "Hey there, thanks for the new content! 🙏 We'll create a preview build and post the link here in a few minutes." >/tmp/comment
+        gh pr comment ${PR_ID} -F /tmp/comment
+      env:
+        GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+    - name: Build container image
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        push: false
+        load: true
+        tags: ovirt-site:latest
+        cache-from: type=gha
+        cache-to: type=gha,mode=max
+    - name: Set up Jekyll cache
+      uses: actions/cache@v1
+      with:
+        path: vendor/bundle
+        key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-gems-
```

Comment:
IIUC this key does not exist and that would give the same result as not specifying `restore-keys`